fragmenter/utils.py

```diff
@@ -213,7 +213,7 @@ def mol_to_tagged_smiles(infile, outfile):
         oechem.OEWriteMolecule(ofs, mol)
 
 
-def create_mapped_smiles(molecule, tagged=True):
+def create_mapped_smiles(molecule, tagged=True, explicit_h=True, isomeric=True):
```

Comment:
Let's favor parameter names like `explicit_hydrogens` instead of abbreviations like `explicit_h` when possible.